components/compliance-service/config/types.go

```diff
@@ -71,7 +71,6 @@ type Manager struct {
 
 // InspecAgent service specific config options
 type InspecAgent struct {
-	Service
```

Comment:
💭 I'm not sure what the correct solution is here. The problem is that the Compliance stuct embedded InspecAgent, and they both embedded Service which caused a conflict, generating a lint error.

Reply:
If it all compiles then we must not be referencing any fields provided by service directly on any InspecAgent structs so this seems reasonable to me.